dags/openshift_nightlies/scripts/install/rosa.sh

```diff
@@ -476,7 +477,13 @@ install(){
         else
             INSTALLATION_PARAMS="${INSTALLATION_PARAMS} --multi-az"  # Multi AZ is default on hosted-cp cluster
         fi
-        rosa create cluster --tags=User:${GITHUB_USERNAME} --cluster-name ${CLUSTER_NAME} --version "${ROSA_VERSION}" --channel-group=${MANAGED_CHANNEL_GROUP} --compute-machine-type ${COMPUTE_WORKERS_TYPE} --replicas ${COMPUTE_WORKERS_NUMBER} --network-type ${NETWORK_TYPE} ${INSTALLATION_PARAMS} ${ROSA_HCP_PARAMS}
+        if [ "$SPOT_WORKERS" == "true" ]; then
+            rosa create cluster --tags=User:${GITHUB_USERNAME} --cluster-name ${CLUSTER_NAME} --version "${ROSA_VERSION}" --channel-group=${MANAGED_CHANNEL_GROUP} --compute-machine-type ${COMPUTE_WORKERS_TYPE} --network-type ${NETWORK_TYPE} ${INSTALLATION_PARAMS} ${ROSA_HCP_PARAMS}
+            _wait_for_cluster_ready ${CLUSTER_NAME}
+            rosa create machinepool -c ${CLUSTER_NAME} --name="spot-pool" --replicas=${COMPUTE_WORKERS_NUMBER} --instance-type=${COMPUTE_WORKERS_TYPE} –label="rosa-spots=true" --use-spot-instances
```

Comment:
Spot instances will not work for HyperShift, we should tackle it this way.

```
rosa create cluster --tags=User:${GITHUB_USERNAME} --cluster-name ${CLUSTER_NAME} --version "${ROSA_VERSION}" --channel-group=${MANAGED_CHANNEL_GROUP} --compute-machine-type ${COMPUTE_WORKERS_TYPE} --network-type ${NETWORK_TYPE} ${INSTALLATION_PARAMS} ${ROSA_HCP_PARAMS}
        if [ "$SPOT_WORKERS" == "true" ]; then
            _wait_for_cluster_ready ${CLUSTER_NAME}
            rosa create machinepool -c ${CLUSTER_NAME} --name="spot-pool" --replicas=${COMPUTE_WORKERS_NUMBER} --instance-type=${COMPUTE_WORKERS_TYPE} –label="rosa-spots=true" --use-spot-instances
        fi

```

Reply:
As discussed can you ensure the machinepool name as 
`${CLUSTER_NAME}-spot-pool` name